verification/curator-service/ui/src/components/SnackbarAlert/index.tsx

```diff
@@ -1,11 +1,11 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import Snackbar from '@material-ui/core/Snackbar';
 import MuiAlert, { AlertProps } from '@material-ui/lab/Alert';
 
 interface SnackbarAlertProps {
-    isOpen: boolean;
+    isOpen: boolean | undefined;
     onClose: (value: boolean) => void;
-    message: string;
+    message: string | ReactNode;
```

Comment:
What was the reason for this change?

Reply:
it is necessary to circumvent this error:
![image](https://user-images.githubusercontent.com/51946947/152542534-af13af4f-f95d-4411-be34-a89e8d996c88.png)